src/test/java/core/basesyntax/strategy/impl/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private OperationHandler operationHandler;
+
+    @Before
+    public void setUp() {
+        operationHandler = new PurchaseOperationHandler();
+    }
+
+    @Test
+    public void purchaseHandler_Ok() {
+        FruitStorage.fruits.put("banana", 100);
```

Comment:
Lets move this FruitStorage filling to the `setUp()` check other handler tests as well